reference/AnimateCtrlGeometry/main.cpp

```diff
@@ -24,7 +26,33 @@ struct AnimateCtrlGeometry : TopWindow {
 			}
 		}
 		
-		SetTimeCallback(1000, [&]{ Animate(ctrls, targets, 300); });
+		ScheduleAnimation();
+	}
+	
+	void ScheduleAnimation()
+	{
+		constexpr int launch_delay_ms = 1000;
+		constexpr int duration_ms = 300;
+		
+		reset.Disable();
+		SetTimeCallback(launch_delay_ms, [=]{
+			Animate(ctrls, targets, duration_ms); reset.Enable();
+		});
+	}
+	
+	void AddResetButton()

```

Comment:
Isn't this a bit overkill for adding a single button? Nothing deal breaking, of course.

Reply:
Fixed! Thanks!